src/tools/Support/scripts/README.md

```diff
@@ -0,0 +1,7 @@
+# Support Scripts #
+Scripts created to be run on the HLH, DVM, ASDK HOST or Jumpbox from an administrative powershell session. These scripts are referenced from within the Release Notes for Azure Stack Integrated System as well as Azure Stack Development Kit (ASDK) to solve or workaround identified Known Issues.
+
+The script scripts will contain information on relevancy for both build and deployment type as well as intended use and overall environmen impact (as needed).
+
+##  Available Support Scripts: ##
+-  Start-ResourceSynchronization.ps1 (applies to both Integraded System and ASDK; relevant build(s): 1802)
```

Comment:
Is it for builds 1802 and above? If its only for 1802, is this info still relevant? I don't suppose we support builds over 3 months old, do we?